e2e/test/E2ETests.csproj

```diff
@@ -2,21 +2,18 @@
   <PropertyGroup>
     <TargetFrameworks>netcoreapp2.1;net472;net451</TargetFrameworks>
     <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp2.1</TargetFrameworks>
-    <!-- TODO #176: Enable warnings as errors. -->
     <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
     <IsPackable>false</IsPackable>
     <RootNamespace>Microsoft.Azure.Devices.E2ETests</RootNamespace>
     <AssemblyName>Microsoft.Azure.Devices.E2ETests</AssemblyName>
-    <!-- Don't auto-compile platform specific files: -->
-    <DefaultItemExcludes>config/**;netstandard20/**;net451/**;$(DefaultItemExcludes)</DefaultItemExcludes>
```

Comment:
Instead of excluding so many files in bulk, remove them selectively below in our target platform-specific sections